migration/sql-files/083-codebases-spaceid-url-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE UNIQUE INDEX codebases_spaceid_url
```

Comment:
just this command won't suffice since we will also need to handle duplicate entries in table otherwise migration will fail.

Reply:
use a `_idx` suffix for the index name (just a convention)

Reply:
@surajssd To delete duplicates you can follow this approach:

```go
-- Delete duplicate links in existence and keep only one
-- See here: https://wiki.postgresql.org/wiki/Deleting_duplicates
DELETE FROM work_item_links
WHERE id IN (
    SELECT id
    FROM (
        SELECT id, ROW_NUMBER() OVER (partition BY link_type_id, source_id, target_id ORDER BY id) AS rnum
        FROM work_item_links
    ) t
    WHERE t.rnum > 1
);
```

It is in our code already: https://github.com/fabric8-services/fabric8-wit/blob/288c2567b20bfbe84cce7db056f24fc53dc4c46f/migration/sql-files/012-unique-work-item-links.sql#L1 .